packages/contracts-watr/.compiler.json

```diff
@@ -0,0 +1,9 @@
+{
+    "version": "0.6.10",
```

Comment:
That probably would not be in spirit of minimal changes - though it may be worth considering bumping up the solc version.